lib/sacastats/utils/utils.ex

```diff
@@ -22,6 +22,10 @@ defmodule SacaStats.Utils do
     numerator / ((denominator == 0 && 1) || denominator)
   end
 
+  def safe_divide(numerator, denominator, to_round) when to_round == 0 do
+    safe_divide(numerator, denominator) |> Float.round(to_round) |> trunc()
+  end
```

Comment:
Would it be better to call `trunc/1` in the template on a case-by-case basis? I feel like `safe_divide/n` should always return a float, instead of returning a float unless `to_round == 0`

Reply:
Yeah okay, that makes sense.